test/chaat/app_test.clj

```diff
@@ -0,0 +1,12 @@
+(ns chaat.app-test
+  (:require [chaat.fixture :as fixture]
+            [clojure.test :refer :all]
+            [clj-http.client :as client]))
+
+(use-fixtures :each fixture/test-fixture)
+
+(deftest end-end-test
+  (testing "get request to health check returns status 200"
+    (let [address "http://0.0.0.0:3010/health-check"
+          response (client/get address)]
+      (is (= 200 (:status response))))))
```

Comment:
suggestion: if you are testing this, would be a good idea to test the negative cases as well. 
eg. If the db connection is not established, it should return an error response

Reply:
Would like to speak to you some more about testing failure cases in end->end tests.